dev-setup.ps1

```
@@ -0,0 +1,275 @@
+Info "Backend: checking conda environment..."
+$BACKEND_ENV = Join-Path $BACKEND_DIR ".env"
+if (Test-Path $BACKEND_ENV) {
+    Skip "Backend conda environment (.env)"
+} else {
+    Info "Backend: creating conda environment with Python 3.12..."
+    conda create -p $BACKEND_ENV python=3.12 -y
+    Ok "Backend conda environment created"
+}
+
+Info "Backend: checking Python packages..."
+$backendHasFastapi = conda run -p $BACKEND_ENV python -c "import fastapi" 2>$null
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'backendHasFastapi' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)